readme.md

```diff
@@ -20,7 +20,7 @@ Like other Yeoman generators, this simply creates the scaffolding of files for y
 - Project
 - Word
 
-Choose to create the Office projects using plain HTML, CSS & JavaScript (*mirroring the same projects that Visual Studio creates*) or create Angular-based projects.
+Choose to create Office Add-in projects using plain HTML, CSS & JavaScript (*mirroring the same projects that Visual Studio creates*) or create Angular-based projects.

```

Comment:
Should this also include react-based projects?

Choose to create Office Add-in projects using plain HTML, CSS & JavaScript (*mirroring the same projects that Visual Studio creates*), create Angular-based projects, or create React-based projects.

Reply:
I don't know. I have no idea about the state of the React templates. I just wonder if it was left out on purpose.